examples/samv71_xplained_ultra/adc/dma/main.cpp

```diff
@@ -0,0 +1,134 @@
+/*
+ * Copyright (c) 2023, Christopher Durand
+ *
+ * This file is part of the modm project.
+ *
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/.
+ */
+
+#include <modm/board.hpp>
+#include <modm/io/iostream.hpp>
+#include <array>
+#include <atomic>
+
+/*
+ * ADC example with DMA and hardware timer trigger.
+ *
+ * Channels 0 and 8 are sampled 5 times in a sequence. The sampling is triggered
+ * by the Timer Counter Channel 0 TIOA output.
+ */
+
+using namespace modm::platform;
+using namespace modm::literals;
```

Comment:
Isn't this already in the board.hpp?

Reply:
In board.hpp we have:
```
namespace Board
{
/// @ingroup modm_board_samv71_xplained_ultra
/// @{
using namespace modm::literals;
using namespace modm::platform;
```

This is much more sensible than having it in global scope by default in the board header.

Reply:
Yeah, we should probably do that. I'd also argue that as a library we should not put things in the global namespace that could lead to issues for users.

The SAM headers are a good example why we shouldn't do that. They define structs and macros with peripheral names in the global namespace. In the DMA driver I've tried to name a class like the peripheral in the data sheet, `Xdmac`. I had to give up because of ambiguities and renamed it `Dma`.

Thankfully Atmel mostly uses such awkward naming that identifiers rarely collide :smile:, like `Afec` for ADC, `Acc` for comparator and `Twihs` for I²C ...